<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>291</width>
      <height>53</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QPushButton" name="pathButton">
       <property name="text">
        <string>Выбрать...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Выберете изображение:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="pathLine">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>63</y>
      <width>102</width>
      <height>334</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="coefficient0">
       <property name="text">
        <string>0,00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="coefficient5">
       <property name="text">
        <string>0,05</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="coefficient10">
       <property name="text">
        <string>0,10</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="coefficient15">
       <property name="text">
        <string>0,15</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="coefficient20">
       <property name="text">
        <string>0,20</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="coefficient25">
       <property name="text">
        <string>0,25</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="coefficient30">
       <property name="text">
        <string>0,30</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="coefficient35">
       <property name="text">
        <string>0,35</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="coefficient40">
       <property name="text">
        <string>0,40</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="coefficient45">
       <property name="text">
        <string>0,45</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="coefficient50">
       <property name="text">
        <string>0,50</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="compressed">
       <property name="text">
        <string>Сжать</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="initialImage">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>621</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="compressedImage">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>360</y>
      <width>621</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="initialSize">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Начальный размер:</string>
    </property>
   </widget>
   <widget class="QLabel" name="comSize">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>После сжатия:</string>
    </property>
   </widget>
   <widget class="QLabel" name="coeff">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Коэффициент сжатия:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>981</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
